src/tools/crossgen2/Common/Compiler/DependencyAnalysis/SortableDependencyNode.cs

```diff
@@ -30,12 +30,9 @@ public abstract class SortableDependencyNode : DependencyNodeCore<NodeFactory>,
         /// If two manage to conflict (which is pretty unlikely), just make a new one...
         /// </remarks>
         public abstract int ClassCode { get; }
-        
+
         // Note to implementers: the type of `other` is actually the same as the type of `this`.
-        public virtual int CompareToImpl(ISortableNode other, CompilerComparer comparer)
```

Comment:
I would prefer this stays like it was - it saves boilerplate overrides in 4 nodes right now. If someone adds a new node that needs this and forgets to override this method, the runtime failure mode is 100% diagnosable and 100% repeatable. This also avoids boilerplate in a couple other nodes on the full AOT side of the compiler.

Reply:
Ok, I'll change it back. The reason I did this was because I thought it would prevent us from forgetting to override `CompareToImpl` at compile time, but that's not true either because any class that inherits from a class that overrides it fails to give a compile time error as well (e.g. all the subtypes of Import).